frontend/device/android/device.lua

```diff
@@ -120,6 +129,8 @@ function Device:init()
                 or ev.code == C.APP_CMD_INIT_WINDOW
                 or ev.code == C.APP_CMD_WINDOW_REDRAW_NEEDED then
                 this.device.screen:_updateWindow()
+            elseif ev.code == C.APP_CMD_PAUSE then
+                self.powerd:beforeSuspend()
```

Comment:
Are you sure you want to turn off light when KOReader activity looses focus? Doesn't this affect to other activities?

Reply:
You're right, I was using that in place of a device sleep event.
On sleep the Nova switches off the backlight and doesn't restore it on wake.
I was using it to get past this line:
https://github.com/koreader/koreader/blob/4005bf69aa5667f520a1e837f9c493874cf192fa/frontend/device/generic/powerd.lua#L88
How about I add a function to `SysfsLight` to read backlight state and use that to implement `AndroidPowerD:frontlightIntensityHW`?

Reply:
Anything that works on onyx and doesn't break other devices is actually an improvement. So feel free to code it.